sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import os
+import time
+import hail as hl
+import logging
+
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+
+from sv_pipeline.utils.common import get_sample_subset, get_sample_remap, get_es_index_name, CHROM_TO_XPOS_OFFSET
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode, get_gene_id
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+WGS_SAMPLE_TYPE = 'WGS'
```

Comment:
`WGS_SAMPLE_TYPE` is a constant for the string `'WGS'` and it is a value of `sample_type`. We use the constant to avoid using the instant value of `'WGS'` everywhere.